templates/master.yaml

```diff
@@ -179,198 +205,31 @@ write_files:
       "metadata": {
         "name": "network-policy.net.alpha.kubernetes.io"
       },
-      "description": "fication for a network isolation policy",
+      "description": "Specification for a network isolation policy",
       "versions": [
         {
           "name": "v1alpha1"
         }
       ]
     }
-- path: /srv/fallback-server-dep.yml
```

Comment:
Unfortunately, I should add :). That's why this is a WIP. We are trying to work out a thorny situation in which we worked on the same code across multiple branches (and repos!).